lib/tmuxinator/window.rb

```diff
@@ -108,7 +108,8 @@ def tmux_window_command_prefix
     end
 
     def tmux_window_name_option
-      name ? "-n #{name}" : ""
+      project_name = project.in_current_session ? "#{project.full_name}:" : ""
+      name ? "-n #{project_name}#{name}" : ""
```

Comment:
I would prefer to drop this change, i.e., for the window to always be named without the prefix.